index.js

```diff
@@ -0,0 +1,98 @@
+const express = require('express');
+const knex = require('knex')({
+    client: 'mysql',
+    connection: {
+        host: '127.0.0.1',
+        port: 3306,
+        user: 'root',
+        password: 'my-secret-pw',
+        database: 'test'
+    }
+});
+
+const mealsRouter = require('./routers/meals');
+const reservationsRouter = require('./routers/reservations');
+
+const app = express();
+const port = 3000;
+
+const sendJSON = (res, status, data) => {
+    res.status(status).json(data);
+};
+
+app.use(express.json());
+
+app.use('/api/meals', mealsRouter);
+app.use('/api/reservations', reservationsRouter);
+
+app.get('/test-connection', async (req, res) => {
```

Comment:
Great to see you are testing your code! In the future these kind of tests can be written in code to prove that your code is correct, but for now I would suggest deleting the endpoint.

Looks like Supertest is a good option for these kinds of tests, https://github.com/ladjs/supertest:
```
const request = require('supertest');
const express = require('express');

const app = express();

app.get('/user', function(req, res) {
  res.status(200).json({ name: 'john' });
});

request(app)
  .get('/user')
  .expect('Content-Type', /json/)
  .expect('Content-Length', '15')
  .expect(200)
  .end(function(err, res) {
    if (err) throw err;
  });
```